@ember/app-tsconfig/tsconfig.json

```diff
@@ -49,10 +49,11 @@
 
     // --- Lint-style rules
 
-    // TypeScript also supplies some lint-style checks; nearly all of them are
-    // better handled by ESLint with the `@typescript-eslint`. This one is more
-    // like a safety check, though, so we leave it on.
-    "noPropertyAccessFromIndexSignature": true,
+    // https://www.typescriptlang.org/tsconfig/#noPropertyAccessFromIndexSignature
+    //
+    // This setting was historically turned on (true) in @tsconfig/ember, 
+    // But it's mostly been unhelpful, and makes type-ergonomics worse.
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
`false` is the default, right? Is there a good reason to leave this here rather than just removing entirely? Setting the setting to its default with a not-super-specific comment doesn't seem that helpful to me, I'd think we would just omit it -- I don't think we need to provide justifications for all the flags that we leave as defaults, unless there's a really Ember-specific reason that they need to be defaults? And your reasoning here has nothing to do with Ember, right, just a general typescript reason?

Reply:
Correct, nothing ember specific. 

I'm good with removing